torch/distributions/gamma.py

```diff
@@ -7,25 +7,10 @@
 from torch.distributions.utils import expand_n
 
 
-class _StandardGamma(Function):
-    @staticmethod
-    def forward(ctx, alpha):
-        x = torch._C._standard_gamma(alpha)
-        ctx.save_for_backward(x, alpha)
-        return x
-
-    @staticmethod
-    @once_differentiable
-    def backward(ctx, grad_output):
-        x, alpha = ctx.saved_tensors
-        grad = torch._C._standard_gamma_grad(x, alpha)
-        return grad_output * grad
-
-
 def _standard_gamma(alpha):
     if not isinstance(alpha, Variable):
         return torch._C._standard_gamma(alpha)
-    return _StandardGamma.apply(alpha)
+    return alpha._standard_gamma()
```

Comment:
I think `torch._C._standard_gamma(alpha)` might work on both `Tensors` and `Variables`

Reply:
doesn't seem to work on Variables; in any case, alpha._standard_gamma() is the right thing to do once we merge Variable and tensor.

Reply:
Could you explain this? As I understand `torch._C._standard_gamma()` draws a random sample (without knowledge of derivatives), whereas `alpha._standard_gamma()` draws a sample with knowledge of derivatives. I'm using the `torch._C._standard_gamma()` without derivatives in #4117, where the derivative needs to be computed in a different way.

Reply:
Did you mean that `alpha._standard_gamma()` should work on both cases?

Reply:
I mean we are going to merge Variables and Tensors (so there is only one case, not both); you'll still be able to whatever you need to (by marking no_grad(), or no_record(), or whatever we call the API).